### ISSUE TITLE
Burst won't install on 7.1.2 ATV

### ISSUE THREAD

**Issue by deleder2k** [elgatito/plugin.video.elementum#215]
Hi. I can't install Burst 0.5.10 on the latest Elementum on my ATV (7.1.2). I've used the Kodi version 18 Alpha 2. There's a pop up every time I launch Elementum, but after clicking yes it says that installation has failed. I've also tried downloading https://burst.surge.sh/release/script.quasar.burst-0.5.10.zip and installing it, but it says "Failed to install Add-on from zip" and "The dependency on plugin.video quasar version 0.0.0.0" and then it fades away. 

I've used the All in one elementum zip file 0.0.58

If anyone knows what's going on it would be fantastic. I've tried reinstalling Kodi and Elementum without luck.

**Comment by elgatito** [comment]
Quasar's Burst is not what you need.
http://elementum.surge.sh/ -> https://github.com/elgatito/script.elementum.burst/releases/download/v0.0.19/script.elementum.burst-0.0.19.zip

You should have Elementum repository installed in your Kodi. If not - you can send logs to find the problem.
https://kodi.wiki/view/Log_file/Easy

**Comment by deleder2k** [comment]
Thanks. It did work. I tried installing the 0.0.59 zip manually now (Arm v7), but it does not add a source. How can I install the repository in Kodi? I've also tried adding many different links, but I can't seem to figure it out.

**Comment by elgatito** [comment]
You should read first and provide logs, that were asked before.

**Comment by deleder2k** [comment]
Oh, sorry. I misread what you wrote. I wasn't sure if the repo would be installed when I installed the zip automatically. 

I did reinstall using the arm7 one - not the universal. Now the repo is installed. Thanks.

LOG: I tried to play Tomb Raider in 2160p. Searched first for it. Kodi crash when I select the stream. This does not always happen, but pretty often. Looks like the problem does not occur If I start the stream remotely.

http://paste.kodi.tv/kacopigabi
